{"name": "test_login_with_invalid_credentials", "status": "failed", "statusDetails": {"message": "AssertionError: assert False\n +  where False = <method-wrapper '__contains__' of str object at 0x0000024ECBDCDE90>('Warning: No match for E-Mail Address and/or Password.ABC')\n +    where <method-wrapper '__contains__' of str object at 0x0000024ECBDCDE90> = 'Warning: No match for E-Mail Address and/or Password.'.__contains__\n +      where 'Warning: No match for E-Mail Address and/or Password.' = <selenium.webdriver.remote.webelement.WebElement (session=\"dc7bbea2b910b5bb6a36f30d5de5fcc0\", element=\"f.7D6CCC9C695417F931FA3316DB46CBDB.d.C6A1CA2A1B2BCEF9C49D81FE7499FAE5.e.61\")>.text\n +        where <selenium.webdriver.remote.webelement.WebElement (session=\"dc7bbea2b910b5bb6a36f30d5de5fcc0\", element=\"f.7D6CCC9C695417F931FA3316DB46CBDB.d.C6A1CA2A1B2BCEF9C49D81FE7499FAE5.e.61\")> = find_element('xpath', \"//div[@id='account-login']/div[1]\")\n +          where find_element = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"dc7bbea2b910b5bb6a36f30d5de5fcc0\")>.find_element\n +            where <selenium.webdriver.chrome.webdriver.WebDriver (session=\"dc7bbea2b910b5bb6a36f30d5de5fcc0\")> = <PYTEST.test_3.test_login.TestLogin object at 0x0000024ECBD47110>.driver\n +          and   'xpath' = By.XPATH", "trace": "self = <PYTEST.test_3.test_login.TestLogin object at 0x0000024ECBD47110>\n\n    @allure.severity(allure.severity_level.BLOCKER)\n    def test_login_with_invalid_credentials(self):\n        self.driver.find_element(By.XPATH, \"//a[@title='My Account']\").click()\n        self.driver.find_element(By.LINK_TEXT, \"Login\").click()\n        self.driver.find_element(By.ID, \"input-email\").send_keys(\"\")\n        self.driver.find_element(By.ID, \"input-password\").send_keys(\"\")\n        self.driver.find_element(By.XPATH, \"//input[@value='Login']\").click()\n    \n        expected_warning_message = \"Warning: No match for E-Mail Address and/or Password.ABC\"\n        # Now error message having not only text include, I tags so i will check contains\n        self.wait = WebDriverWait(self.driver, 10)\n        self.element = self.wait.until(EC.presence_of_element_located((By.XPATH, \"//div[@id='account-login']/div[1]\")))\n    \n        allure.attach(self.driver.get_screenshot_as_png(), name=\"login_without_entering_credentials\",\n                      attachment_type=AttachmentType.PNG)\n    \n>       assert self.driver.find_element(By.XPATH, \"//div[@id='account-login']/div[1]\").text.__contains__(\n            expected_warning_message)\nE       assert False\nE        +  where False = <method-wrapper '__contains__' of str object at 0x0000024ECBDCDE90>('Warning: No match for E-Mail Address and/or Password.ABC')\nE        +    where <method-wrapper '__contains__' of str object at 0x0000024ECBDCDE90> = 'Warning: No match for E-Mail Address and/or Password.'.__contains__\nE        +      where 'Warning: No match for E-Mail Address and/or Password.' = <selenium.webdriver.remote.webelement.WebElement (session=\"dc7bbea2b910b5bb6a36f30d5de5fcc0\", element=\"f.7D6CCC9C695417F931FA3316DB46CBDB.d.C6A1CA2A1B2BCEF9C49D81FE7499FAE5.e.61\")>.text\nE        +        where <selenium.webdriver.remote.webelement.WebElement (session=\"dc7bbea2b910b5bb6a36f30d5de5fcc0\", element=\"f.7D6CCC9C695417F931FA3316DB46CBDB.d.C6A1CA2A1B2BCEF9C49D81FE7499FAE5.e.61\")> = find_element('xpath', \"//div[@id='account-login']/div[1]\")\nE        +          where find_element = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"dc7bbea2b910b5bb6a36f30d5de5fcc0\")>.find_element\nE        +            where <selenium.webdriver.chrome.webdriver.WebDriver (session=\"dc7bbea2b910b5bb6a36f30d5de5fcc0\")> = <PYTEST.test_3.test_login.TestLogin object at 0x0000024ECBD47110>.driver\nE        +          and   'xpath' = By.XPATH\n\ntest_login.py:41: AssertionError"}, "attachments": [{"name": "login_without_entering_credentials", "source": "91428eb0-81d4-4ec7-adbf-ac577a8dcd51-attachment.png", "type": "image/png"}], "start": 1761483865200, "stop": 1761483866589, "uuid": "88978c6a-be23-4d8d-bdec-ccdef2fa9932", "historyId": "053d68a8a3c7e7c127b66b08a447a3ee", "testCaseId": "053d68a8a3c7e7c127b66b08a447a3ee", "fullName": "test_login.TestLogin#test_login_with_invalid_credentials", "labels": [{"name": "severity", "value": "blocker"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-4222FVV"}, {"name": "thread", "value": "9124-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login"}], "titlePath": ["test_login.py", "TestLogin"]}